xbmc/guilib/DDSImage.cpp

```diff
@@ -120,12 +119,12 @@ bool CDDSImage::Create(const std::string &outputFile, unsigned int width, unsign
 {
   if (!brga)
     return false;
-  if (!Compress(width, height, pitch, brga, maxMSE))
-  { // use ARGB
-    Allocate(width, height, XB_FMT_A8R8G8B8);
-    for (unsigned int i = 0; i < height; i++)
-      memcpy(m_data + i * width * 4, brga + i * pitch, std::min(width * 4, pitch));
-  }
+
+  // use ARGB
+  Allocate(width, height, XB_FMT_A8R8G8B8);
+  for (unsigned int i = 0; i < height; i++)
+    memcpy(m_data + i * width * 4, brga + i * pitch, std::min(width * 4, pitch));
+
```

Comment:
The stuff that writes DDS should be completely removed if this goes in (prob in TextureCache). The disk usage will go up a lot otherwise (it's just storing a 32bpp image directly at this point), and I doubt there's much time-decode benefit on any platform that is supported.